hack/generators/release-controllers/content/osd_rc_deployments.py

```diff
@@ -157,7 +157,7 @@ def _add_osd_rc_deployment(gendoc):
             'namespace': context.config.rc_deployment_namespace,
         },
         'spec': {
-            'replicas': 1,
```

Comment:
This is E4 in the doc.
I made a separate PR https://github.com/openshift/release/pull/14400
Easier to revert for E12 (scale up).

Reply:
Right, but these changes must reflect the same state (replicas: 0) at that point in time or the release-controllers will all scale back up when these changes merge.

Reply:
The alternative is to use this PR as the Scale Up step (E12).

Reply:
If scale down has to go with this PR.
Closing 14400.